makefiles/lf.mk

```diff
@@ -3,31 +3,37 @@ $(error Use the main Makefile)
 endif
 
 SUBPROJECTS += lf
-LF_VERSION  := r22
+LF_VERSION  := r24
 DEB_LF_V    ?= 0~$(LF_VERSION)
 
 lf-setup: setup
 	$(call GITHUB_ARCHIVE,gokcehan,lf,$(LF_VERSION),$(LF_VERSION))
 	$(call EXTRACT_TAR,lf-$(LF_VERSION).tar.gz,lf-$(LF_VERSION),lf)
-	mkdir -p $(BUILD_STAGE)/lf/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/{bin,share/man/man1}
+	mkdir -p $(BUILD_STAGE)/lf/{etc/profile.d,$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/{bin,share/{lf,man/man1,zsh/site-functions,vim/vimfiles/syntax}}}
+
+ifneq (,$(findstring darwin,$(MEMO_TARGET)))
+LF_DEPS := lf-setup
+else
+LF_DEPS := lf-setup libiosexec
```

Comment:
don't add libiosexec as a dep, it is useless.

Reply:
It was added since everything was linked to it (through a huge commit); I'll remove it shortly.